eng/Versions.props

```diff
@@ -30,6 +30,7 @@
       release, increase this number by one.
     -->
     <NETStandardPatchVersion>0</NETStandardPatchVersion>
+    <MicrosoftNetCompilersToolsetVersion>3.9.0-2.20516.4</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This really needs a comment, an issue tracking removal of this line, or something similar.

Reply:
Comment added